react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -1209,7 +1252,8 @@ function _mapStateToProps(state, ownProps): Object {
         _videoObjectPosition: getVideoObjectPosition(state, participant?.id),
         _videoTrack,
         ...size,
-        _gifSrc: mode === 'chat' ? null : gifSrc
+        _gifSrc: mode === 'chat' ? null : gifSrc,
+        sourceNameSignalingEnabled
```

Comment:
nit: can we rename this to _sourceNameSignalingEnabled to follow the convention of the other props from mapStateToProps?

Reply:
Changed, though I don't think it's a good convention.